api/test/scenarios-pu-data/scenarios-pu-data-service.integration.e2e-spec.ts

```diff
@@ -29,7 +29,11 @@ describe(`scenarios-pu-data fetch`, () => {
     count = (await world.GivenScenarioPuDataExists()).length;
 
     // Act
-    result = await service.findAll();
+    result = await service.findAll(undefined, {
```

Comment:
This test was not working (but we couldn't find it on CI - it was returning all 20k ~ rows ;)